src/app/containers/ATIAnalytics/params/radioPage/buildParams.js

```diff
@@ -15,9 +15,17 @@ export const buildRadioATIParams = (
 
   const { id, language, pageTitle, pageIdentifier, contentType } = pageData;
 
+  const isLiveRadio = contentType === 'player-live';
+
+  const getContentId = (assetType) => {
+    const guid = id.split('/').pop();
+    const contentId = `urn:bbc:${assetType}:`.concat(guid);
+    return contentId;
+  };
+
   return {
     appName: atiAnalyticsAppName,
-    contentId: id,
+    contentId: isLiveRadio ? id : getContentId('pips'),
```

Comment:
Could the param `pips` be hardcoded inside `getContentId` (only if that function is not used anywhere else)?